src/modules/ShareActionHandlerModule.ts

```diff
@@ -0,0 +1,17 @@
+import {NativeModules} from 'react-native';
+
+const {ShareActionHandlerModule} = NativeModules;
+
+type ShareActionContent = {
+    id: string;
+    content: string;
+    mimeType: string;
+    processedAt: string;
+    aspectRatio: number;
+};
+
+type ShareActionHandlerType = {
+    processFiles(callback: (array: ShareActionContent[]) => void): void;
+};
+
+export default ShareActionHandlerModule as ShareActionHandlerType;
```

Comment:
```suggestion
export default ShareActionHandlerModule;
```

Reply:
Instead of asserting, please add the type as the rest of native modules (`src/types/modules/react-native.d.ts`):
```ts
    interface NativeModulesStatic {
        BootSplash: BootSplashModule;
        HybridAppModule: HybridAppModule;
        StartupTimer: StartupTimer;
        RNTextInputReset: RNTextInputResetModule;
        RNNavBarManager: RNNavBarManagerModule;
        EnvironmentChecker: EnvironmentCheckerModule;
        ShortcutManager: ShortcutManagerModule;
    }
```